roles/sriov/tasks/uninstall.yml

```
@@ -0,0 +1,12 @@
+---
+
+# TODO: running on master node for now (fix to run on multiple nodes)
+
+- name: remove sriov plugin files in /tmp
+  file:
+    path: '{{ item }}'
```

To follow `OnRamp`s usage of quotation marks
```suggestion
    path: "{{ item }}"
```

---

code removed, not applicable.